src/App.tsx

```diff
@@ -1,31 +1,43 @@
 import './App.css';
 import React from 'react';
+import { useState } from 'react';
+
 import GraphEditor from './components/GraphEditor';
-import Canvas from './components/Canvas';
-import mapGraph from './logic/graphMapping';
+import CanvasAnimator from './components/CanvasAnimator';
+
 import { Graph } from './logic/commonInterfaces';
-import { useState } from 'react';
 import samples from './logic/graphSamples';
+import disassembleGraph from './logic/graphDisassembling';
+import createFrames from './logic/graphFrames';
+import mapGraph from './logic/graphMapping';
+import labelGraph from './logic/graphLabeling';
+import { DijkstraTracer } from './logic/dijkstra';
 
 function App() {
-  const [mappedGraph, setMappedGraph] = useState({
-    graph: mapGraph(samples.simple.graph), //TODO handle initial state better
-    path: [],
-  });
+  const processGraph = (graph: Graph) => {
+    const disassembledGraph = disassembleGraph(graph);
+    const dijkstra = new DijkstraTracer(graph);
+    const frames =
+      createFrames(disassembledGraph, dijkstra.paths, dijkstra.traversalHistory);
```

Comment:
this looks like an error (passing `dijkstra` would be enough and save the space)

line-length of 90 is really too radical